### PR TITLE
Issue #8544: format xml sources (xdocs non-checks)

### PR DESCRIPTION
**Pull request by pbludov**
Issue #8544

Checkstyle all xdoc files (except the checks) were formatted with two spaces indented.

There is a questionable one: `property_types.xml`. We should discuss this.
To satisfy the rules, all `<b>}</b>` tags should be indented __more__ than the enclosing `<pre>` tag.

There is at least three options:

* remove `<b></b>` and keep only the comments
* add this file to the exception list (current)
* wrap long lines

Formatting for checks will be in in the next PR(s).

### CODE REVIEW

File: src/xdocs/property_types.xml [hunk 1]
```diff
@@ -714,12 +714,12 @@
...
-    <b>}</b>
-    finally {
-        ...
-    <b>}</b>
+                  try {
+                      ...
+                  <b>}</b>
+                  finally {
+                      ...
+                  <b>}</b>
```
Comment (romani) [inline src/xdocs/property_types.xml:722]: Please recheck web site, this might make rendering a bit too much right indented

**Comment by pbludov** [comment]
New render of RightCurlyCheck
image

File: src/xdocs/property_types.xml (hunk 1 above)
Comment (pbludov) [reply src/xdocs/property_types.xml]: this is what i'm talkin about. The xml fomatter wants `<b>}</b>` to be indented with 18 spaces.
From its point of view, the document looks like
```xml
              <div class="wrapper">
                <pre>
    <b>}</b>
    <b>}</b>
                </pre>
```
with some text inside. And it think that `<b>` tag should be moved to right.

**Comment by pbludov** [comment]
> I recommend to move updates in PRE in separate PR. To merge simple stuff quicker.

Done. Now this file is excluded.

**Comment by romani** [comment]
>New render of RightCurlyCheck

Not good.
Compare with https://checkstyle.org/config_blocks.html#RightCurly_Examples

**Pull request merged**